docs/Developers/Development_Setup.rst

```diff
@@ -43,14 +43,25 @@ Using virtualenv
 - Install virtualenv using pip :
     ::
 
-        $ pip install virtualenv
+        $ pip3 install virtualenv
 
 - Create the virtualenv :
     ::
 
         $ cd working_dir # move into the dir where you want to create coala-venv
         $ virtualenv coala-venv
 
+-NOTE:
+ If you have both Python 3 and Python 2 installed check this out first.
+
+::
+
+    $ virtualenv coala-venv -p $(which python3)
+
+-This creates an isolated Python 3 environment called coala-venv
+ in your current directory.
+ As coala only works for Python >= 3.4.
```

Comment:
@jack17529 any reason this can't be discussed here? PMs are only for issues that require privacy and especially not technical issues.